### PR TITLE
Minor cherry-pick merges to 1.12

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

**Comment by lrknox** [comment]
I made this a draft for now, because I want develop tests to run before merging to 1.12

File: tools/test/h5dump/testh5dump.sh.in [hunk 1]
```diff
@@ -183,6 +183,16 @@ $SRC_H5DUMP_TESTFILES/tvms.h5
 $SRC_H5DUMP_TESTFILES/err_attr_dspace.h5
 $SRC_H5DUMP_TESTFILES/tCVE_2018_11206_fill_old.h5
 $SRC_H5DUMP_TESTFILES/tCVE_2018_11206_fill_new.h5
+<<<<<<< HEAD
```
Comment (byrnHDF) [inline tools/test/h5dump/testh5dump.sh.in]: Conflict markers!

**Review by byrnHDF — CHANGES_REQUESTED** [review]
Conflict markers in testh5dump.sh.in need clean up.

**Comment by lrknox** [comment]
Conflict resolved and references to onion vfd and test files removed.

File: tools/test/h5dump/testh5dump.sh.in (hunk 1 above)
Comment (lrknox) [reply tools/test/h5dump/testh5dump.sh.in]: fixed.

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**